blob/example_test.go

```diff
@@ -335,7 +337,15 @@ func ExampleOpen() {
 	}
 	fmt.Println("Got expected error opening a nonexistent path")
 
-	if _, err := blob.Open(ctx, "file://"+dir); err != nil {
+	// Ensure the path has a leading slash; fileblob ignores the URL's
+	// Host field, so URLs should always start with "file:///". On
+	// Windows, the leading "/" will be stripped, so "file:///c:/foo"
+	// will refer to c:/foo.
+	urlpath := url.PathEscape(filepath.ToSlash(dir))
```

Comment:
Should we handle converting separators to '/' in package fileblob instead of letting user to it? Is "c:\foo\bar" invalid?

Reply:
No; according to [this](https://blogs.msdn.microsoft.com/ie/2006/12/06/file-uris-in-windows/), even on Windows, URLs referring to files should use "/" as a path separator. So doing this on the user side is correct.

That said, it might work either way; i.e., if they pass `file:///c:\foo\bar.txt`, it might work. "\" is definitely illegal in the host part, which is why the test was failing before, but it might be legal in the path part, and we are just passing the path through to `OpenBucket`. I don't want to document it as working though., and the example should do the correct thing.